query_builders/query_builder.py

```diff
@@ -311,11 +313,21 @@ def list_where_rel_conditions_per_dict(mp: dict) -> ([], {}):
                 leftdir = (check[-1] if check[-1] in ['<'] else '')
                 rightdir = (check[-1] if check[-1] in ['>'] else '')
                 inner_cond_list = []
+                matched_lst=[]
                 assert isinstance(content, dict)
                 for operator_, lst in content.items():
                     assert operator_ in ['include', 'exclude', 'include_matched', 'exclude_matched']
-                    lst_matched_str = "[" + ", ".join(['`' + item + '`' for item in lst]) + "]"
-                    lst_str = "(" + " OR ".join(['x:`' + item + '`' for item in lst]) + ")"
+                    for item in lst:
+                        if re.search(r"\b" + re.escape('uri') + r"\b", item):
```

Comment:
this is supposed to support any key, not just 'uri'
e.g. it could be 
{'Test' : {'rdfs:label': ['SYSBP']}}